apps/studio/components/interfaces/Storage/VectorBuckets/index.tsx

```diff
@@ -157,18 +157,23 @@ export const VectorsBuckets = () => {
                     const created = +bucket.creationTime * 1000
 
                     return (
-                      <TableRow key={id} className="relative cursor-pointer h-16">
+                      <TableRow
+                        key={id}
+                        className={cn('relative cursor-pointer h-16', 'inset-focus')}
+                        onClick={(event) => handleBucketNavigation(name, event)}
+                        onKeyDown={(event) => {
+                          if (event.key === 'Enter' || event.key === ' ') {
+                            event.preventDefault()
+                            handleBucketNavigation(name, event)
+                          }
+                        }}
+                        tabIndex={0}
+                      >
                         <TableCell className="w-2 pr-1">
                           <BucketIcon size={16} className="text-foreground-muted" />
                         </TableCell>
                         <TableCell>
                           <p className="whitespace-nowrap max-w-[512px] truncate">{name}</p>
-                          <button
-                            className={cn('absolute inset-0', 'inset-focus')}
```

Comment:
I think this absolutely positioned button was causing Safari stacking context issues.